coordinator/src/models/hl7-v3/hl7-v3-datatypes-codes.ts

```diff
@@ -160,21 +160,33 @@ export class NhsNumber extends Identifier<"2.16.840.1.113883.2.1.4.1"> {
   }
 }
 
-export class SdsUniqueIdentifier extends Identifier<"1.2.826.0.1285.0.2.0.65"> {
+export class AgentPersonIdCode extends Identifier<"1.2.826.0.1285.0.2.1.54"> {
   constructor(extension: string) {
-    super("1.2.826.0.1285.0.2.0.65", extension)
+    super("1.2.826.0.1285.0.2.1.54", extension)
   }
 }
 
-export class SdsRoleProfileIdentifier extends Identifier<"1.2.826.0.1285.0.2.0.67"> {
+export class PrescribingCode extends AgentPersonIdCode {
```

Comment:
Is there any reason to have these two subclasses given they're exactly the same? I'd be tempted to replace all uses with the superclass.

Reply:
They have meaning in the domain, intention was to expose these conceptually as there's logic to fall back from prescribing to professional code, yes they have the same oid, but the logic to select a prescribing code vs professional is clearer when they are different classes